src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -610,6 +610,7 @@ private function getWriteInfo(string $class, string $property, $value): Property
             'enable_magic_methods_extraction' => $this->magicMethodsFlags,
             'enable_constructor_extraction' => false,
             'enable_adder_remover_extraction' => $useAdderAndRemover,
+            'value' => $value,
```

Comment:
Is this change needed?

Reply:
Yes, The PropertyInfo needs to know if the value is null or not.

Reply:
Can we add a test covering this case then?

Reply:
Tests are added!